src/components/HomeCrud/HomeForm.js

```diff
@@ -3,22 +3,22 @@ import PropTypes from 'prop-types'
 import * as yup from 'yup'
 
 // FPCC
-import { DOC_IMAGE } from 'common/constants'
+import { IMAGE } from 'common/constants'
 import Form from 'components/Form'
 import { definitions } from 'common/utils/validationHelpers'
 import useEditForm from 'common/hooks/useEditForm'
 
 function HomeForm({ cancelHandler, dataToEdit, submitHandler }) {
   const validator = yup.object().shape({
     logoId: definitions.uuid(),
-    topBackgroundImageId: definitions.uuid(),
-    videoId: definitions.uuid(),
+    bannerImageId: definitions.uuid(),
+    bannerVideoId: definitions.uuid(),
   })
 
   const defaultValues = {
     logoId: '',
-    topBackgroundImageId: '',
-    topBackgroundVideoId: '',
+    bannerImageId: '',
```

Comment:
I may be wrong but this can be `bannerImage` and `bannerVideo` instead of `bannerImageId` and `bannerVideoId`.